tests/test_wsireader.py

```diff
@@ -961,7 +961,7 @@ def test_openslide_mpp_from_tiff_resolution(_sample_svs):
 def test_VFReader():
```

Comment:
It should be test_VirtualWSIReader

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/8387736e723a92110cc719baf0490d46a6518d82